src/components/Form.jsx

```diff
@@ -109,14 +141,18 @@ const Form = () => {
               onChange={handleChange}
             />
           </div>
+          <div>
+            {emailExist && <div>User with this email already exist</div>}
+          </div>
         </div>
-        <button
+        <Button
           type="submit"
-          className="text-white bg-gradient-to-r from-cyan-500 via-cyan-600 to-cyan-800 hover:bg-gradient-to-br focus:ring-4 focus:outline-none focus:ring-blue-300 dark:focus:ring-blue-800 font-medium rounded-lg text-sm px-5 py-2.5 text-center me-2 mb-2"
+          className="text-white bg-gradient-to-r from-cyan-500 via-cyan-600 to-cyan-800 cursor-pointer hover:bg-gradient-to-br focus:ring-4 focus:outline-none focus:ring-blue-300 dark:focus:ring-blue-800 font-medium rounded-lg text-sm px-5 py-2.5 text-center me-2 mb-2"
+          isDisable={emailExist}
         >
           Submit
-        </button>
-        <button
+        </Button>
+        <Button
           className="relative inline-flex items-center justify-center p-0.5 mb-2 me-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-cyan-700 to-blue-300 group-hover:from-cyan-300 group-hover:to-blue-500 hover:text-white dark:text-white focus:ring-4 focus:outline-none focus:ring-cyan-200 dark:focus:ring-cyan-800"
```

Comment:
Remove `className` prop for common components and add `variant` prop to accommodate styles.